code/controllers/subsystem/machines.dm

```diff
@@ -82,6 +82,21 @@ SUBSYSTEM_DEF(machines)
 	if (!resumed)
 		for(var/datum/powernet/powernet as anything in powernets)
 			powernet.reset() //reset the power state.
+		current_part = SSMACHINES_MACHINES
+		src.currentrun = processing.Copy()
+
+	//Processing machines, directly wired to the network.
```

Comment:
This is inaccurate. This covers every single machine that will get processed and aren't in the other lists.